src/HSA/libhsa_enums.jl

```diff
@@ -220,6 +221,22 @@ end
     AGENT_INFO_EXTENSIONS = 20
     AGENT_INFO_VERSION_MAJOR = 21
     AGENT_INFO_VERSION_MINOR = 22
+    AMD_AGENT_INFO_CHIP_ID = 40960
```

Comment:
I think it's going to be far easier to just look at the diffs when the hsa headers get updated and paste in updates (as done here), rather than hoping to auto-regenerate the wrappers at each patch. The underlying API changes only slowly, and the problem we run into is that generated functions often need (and have needed) manual clean-up for things that `Clang.jl` just isn't equipped to handle. (Common example: should `float *foo` be `Array{Float32}` `Ref{Float32}` `Ptr{Float32}` depending on context it could be any of the above)

Reply:
I initially assumed what @wsphillips is saying.

The main reason I merged the two enums was so that the type signature for the functionally identical `getinfo()` function would be the same, otherwise we would have to duplicate, or at least modify, the wrapper for `hsa_agent_get_info()` as well to accept both types.

Reply:
:+1: